src/Compilers/CSharp/Portable/Symbols/AssemblySymbol.cs

```diff
@@ -855,7 +853,7 @@ internal NamedTypeSymbol GetPrimitiveType(Microsoft.Cci.PrimitiveTypeCode type)
             DiagnosticBag? warnings = null, // this is set to collect ambiguity warning for well-known types before C# 7
             bool ignoreCorLibraryDuplicatedTypes = false)
         {
-            // Type from this assembly always wins.
+            // Type from this assembly usually wins unless we search for well-known type and find file-local declaration in source.
```

Comment:
>// Type from this assembly usually wins unless we search for well-known type and find file-local declaration in source. [](http://example.com/codeflow?start=12&length=119)

Not sure about this comment change. Especially that, given the right input, this function should find a file-local type